### ISSUE TITLE
undermineralised skull

### ISSUE THREAD

**Issue by azankl** [obophenotype/human-phenotype-ontology#7378]
**Preferred term label:**
undermineralised skull

**Synonyms**
hypomineralised skull
poorly mineralised skull

**Definition (free text, please give PubMed ID)**
A hypomineralised skull castes little or no acoustic shadow on prenatal ultrasound. This makes visualisation of the intracranial anatomy clearer than normal. In conditions associated with profound hypomineralisation, the skull shape can be distorted by pressure from the ultrasound transducer later in pregnancy.
https://thefetus.net/content/osteogenesis-imperfecta-4/
[PMID: 31173381](https://pubmed.ncbi.nlm.nih.gov/31173381/)

**Parent term (use hpo.jax.org/app)**
Decreased calvarial ossification HP:0005474

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
osteogenesis imperfecta type II [OMIM 166210](https://www.omim.org/entry/166210)
osteogenesis imperfecta type III [OMIM 259420](https://www.omim.org/entry/259420)

**Your nano-attribution (ORCID)**
Andreas Zankl 0000-0001-8612-1062

**Comment by azankl** [comment]
This is very similar to 'decreased calvarial ossification HP:0005474', I wanted to emphasise the prenatal ultrasound appearance. Maybe we should just add the prenatal definition to the existing term?

**Comment by pnrobinson** [comment]
Let's discuss this -- we are planning one more prenatal workshop by zoom.

**Comment by pnrobinson** [comment]
I think that the existing term is a synonym. I will add your above comments for the prenatal appearance.
